modules/appeals_api/app/swagger/v0/appeals.yml

```diff
@@ -47,8 +47,8 @@ paths:
     get:
       tags:
         - Appeals Status
-      summary: Retrieve appeals status for the Veteran with the supplied SSN
-      description: Returns a list of all known appeal records for the given veteran. Includes details of each appeal's current status, priority, and history of updates.
+      summary: Retrieve appeals status for the Veteran with the supplied ICN
```

Comment:
This one should say `SSN` since this is the old v0 doc where we still expect `X-VA-SSN`

Reply:
Thank you for catching that! Was definitely an oversight on my part.

Reply:
Fixed now!